src/Thinktecture.Relay.Server/Maintenance/StatisticsOptions.cs

```diff
@@ -21,4 +21,9 @@ public class StatisticsOptions
 	/// Indicates whether to clean up stale or closed connections or not.
 	/// </summary>
 	public bool EnableConnectionCleanup { get; set; } = false;
+
+	/// <summary>
+	/// The time interval in which the connection's last seen time will be updated.
+	/// </summary>
+	public TimeSpan LastConnectionSeenUpdateInterval { get; set; } = TimeSpan.FromMinutes(1);
```

Comment:
This naming is quite unclear, maybe it should be named `ConnectionLastSeenUpdateInterval` instead.

Reply:
Where is the rename?